en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md

```diff
@@ -33,20 +34,23 @@ In this tutorial, you will be importing an OpenAPI specification and creating a
     !!! tip
         Once you create a project successfully, you are taken to the components landing page. You can also select the components icon from the left menu and open the **Components** landing page. 
 
-4. Click the **REST API Proxy** card.
+4. Click **+Create** on the **Components** landing page. 
```

Comment:
```suggestion
4. On the **Components** landing page, click **Create**.
```

Suggesting this to adhere to this guideline in the Google Development guide:

https://developers.google.com/style/procedures#steps-that-say-where-to-complete-a-task